src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormServiceImpl.java

```diff
@@ -197,7 +197,8 @@ private Form getEditorForm(ExtendedNodeType primaryNodeType, JCRNodeWrapper exis
                     for (Field field : fieldSet.getFields()) {
                         // Set field label and description if not set
                         field.initializeLabel(uiLocale, site, primaryNodeType);
-                        field.setVisible((field.isHide() == null || !field.isHide()) &&
+                        // j:bindedComponent property is normally hidden, but we want to have it in the form in order to choose component to bind
+                        field.setVisible((field.isHide() == null || !field.isHide() || field.getName().equals("j:bindedComponent")) &&
```

Comment:
I would prefer this to be done with json override instead of embedding specific logic in the code. It seems we already have a pattern of doing that as in the case here https://github.com/Jahia/jcontent/blob/master/src/main/resources/META-INF/jahia-content-editor-forms/forms/jmix_conditionalVisibility.json#L14

Reply:
Unless I'm missing something, I was able to show it with json override:

<img width="872" alt="image" src="https://github.com/user-attachments/assets/ff3498ed-62cd-45dd-9962-5f99c13ace28" />

```
{
  "name": "jmix:bindedComponent",
  "priority": 1.0,
  "fields" : [
    {
      "rank": 0.0,
      "name": "j:bindedComponent",
      "hide": false
    }
  ]
}

```

Reply:
Mondays :).

Btw, I don't know if we'd also want to change the labels from the default values, and I think that's something we can also include in the override.